src/Akeneo/Pim/Enrichment/Component/Product/Comparator/Attribute/OptionsComparator.php

```diff
@@ -29,22 +29,26 @@ public function __construct(array $types)
      */
     public function supports($type)
     {
-        return in_array($type, $this->types);
+        return \in_array($type, $this->types);
     }
 
     /**
      * {@inheritdoc}
      */
     public function compare($data, $originals)
     {
+        if (!\is_array($data['data']) || \count($data['data']) !== \count(\array_filter($data['data'], 'is_string'))) {
+            return $data;
+        }
```

Comment:
```suggestion
        try {
            Assert::isArray($data['data']);
            Assert::allIsString($data['data'];
        } catch (\InvalidArgumentException) {
            return $data;
        }
```
(same, but easier to understand IMO)